vllm/config.py

```diff
@@ -964,8 +965,9 @@ def verify_with_model_config(self, model_config: ModelConfig):
                 "awq", "gptq"
         ]:
             # TODO support marlin and squeezellm
+            logger_data = {"quantization": model_config.quantization}
```

Comment:
I've added an `event` suggestion
```suggestion
            logger_data = {"event": "quantization-not-tested", "quantization": model_config.quantization}
```

@robertgshaw2-neuralmagic , I'm not sure this is worth adding extra data for. Thoughts? On one hand, it should be cheap, but I'm not sure how much utility it offers an enterprise